rozy/src/installers/shell.rs

```diff
@@ -59,7 +60,7 @@ impl Installer for Shell {
         download_to(&file, &self.url)?;
 
         let mut command = std::process::Command::new("/bin/bash");
-
+        command.stdout(std::process::Stdio::piped());
```

Comment:
e.g. can this not be something like command.stdout(std::process::stderr) or something (not near a proper computer to look that up)